ftest.sh

```diff
@@ -238,7 +240,7 @@ rm -f core.* *_results.xml
 
 # now run it!
 export PYTHONPATH=./util:../../utils/py/:./util/apricot
-if ! ./launch.py -s \"$TEST_TAG\"; then
+if ! ./launch.py -s "${TEST_TAG_ARR[*]}"; then
```

Comment:
(lint) This is actually an end quote, but due to next char it looks suspect. [SC1079]
(lint) The surrounding quotes actually unquote this. Remove or escape them. [SC2027]